client/src/App.jsx

```diff
@@ -27,21 +31,38 @@ class App extends Component {
       .then(response => response.json())
       .then(jsonResponse => this.setState({
         videos: jsonResponse,
-        selected: jsonResponse[2]
+        selected: jsonResponse[0],
+        selectedIdx: 0
       }));
   }
 
   handleRating(newState, rating) {
     this.setState({
       videos: newState,
-      selected: newState[2],
+      selected: this.state.selected,
       rated: rating
     })
```

Comment:
I'd recommend installing a linter immediately so that your tooling can point out minor issues (eg, missing semicolon) rather than your PR reviewers!